### DISCUSSION TITLE
circleci CI/CD

### DISCUSSION THREAD

**Discussion by csalmon1** [curl/curl#8563] [General]
Hi all. I want to replicate curl's CI/CD pipeline for a research project. In the _.circleci/config.yml_ file I only see executors and images for ubuntu, nothing for windows. Are windows builds handled in a different file, or with a different CI/CD pipeline tool?

**Comment by bagder** [comment]
Run `./scripts/cijobs.pl` in the curl repository to get a full listing with details from all current CI jobs.

We don't run any Windows jobs on Circle CI, no.

**Comment by dfandrich** [comment]
curl uses many CI/CD services. There's a description about a tool recently
written that you can use to summarize them here:
https://curl.se/mail/lib-2022-02/0081.html

**Comment by csalmon1** [comment]
Great info - thanks!
